pkg/latticestore/latticestore.go

```diff
@@ -392,6 +402,11 @@ func (ds *LatticeDataStore) SetTargetGroupByBackendRef(name string, routeName st
 
 }
 
+// TODO: make this DelTargetGroup() method to be private, create seperate methods:
```

Comment:
What's the purpose of these TODO items?

Reply:
Current we use this way to delete a tg cache record that this k8sSerivce backendref in a httproute: 			

`t.latticeDataStore.DelTargetGroup(resTargetGroup.Spec.Name, resTargetGroup.Spec.Config.K8SHTTPRouteName, false)`


we use this way to delete a tg cache record for which this serviceImport backendref in a httproute: 
`t.latticeDataStore.DelTargetGroup(resTargetGroup.Spec.Name, "", true)`		


And we use this way to delete a tg cache record for serviceExport: 
`t.latticeDataStore.DelTargetGroup(resTargetGroup.Spec.Name, "", false)`		


I think these functions are really confusing, we should improve them

Reply:
please remove these TODO comments and create an issue instead